td-loader/fuzz/fuzz_targets/elf.rs

```diff
@@ -0,0 +1,115 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use td_loader::{elf, elf64::*};
+
+fn fuzz_elf_loader(data: &[u8]) {
+    if elf::is_elf(data) {
```

Comment:
How about inverting the logic to remove one level of indentation:

```suggestion
    if !elf::is_elf(data) {
        return;
    }
    
    // ...
```

Same comment for `fuzz_pe_loader()`.

Reply:
Done